conversion.go

```diff
@@ -1,7 +1,82 @@
 // Package conversion TODO(kkweon): Write description
 package conversion
 
+import (
+	"encoding/binary"
+	"errors"
+	"math"
+)
+
+type Endian int
+
+const BigEndian Endian = 0
+const LittleEndian Endian = 1
+
+type Option struct {
+	Endian Endian
+}
+
 // IntToFloat32 converts int to float32
 func IntToFloat32(x int) (float32, error) {
 	return float32(x), nil
 }
+
+// Float32ToBytes converts float32 to []byte with length 4
+func Float32ToBytes(x float32, o *Option) ([]byte, error) {
```

Comment:
에러가 쓰이지 않는다면 굳이 필요 없을 것 같습니다. 필요하기 전에 추가할 필요는 없죠.

```suggestion
func Float32ToBytes(x float32, o *Option) []byte {
```

Reply:
맞는 말씀이네요. 신나게 배웁니다. 음. 근데 이건 왜 Commit suggestion 이 활성화가 안되지?

Reply:
맞는 말씀이네요